nested-spire/nestedA/server/server.conf

```diff
@@ -0,0 +1,34 @@
+server {
+	bind_address = "0.0.0.0"
+	bind_port = "8081"
+	registration_uds_path = "/tmp/spire-registration.sock"
+	trust_domain = "example.org"
+	data_dir = "/opt/spire/data/server"
+	log_level = "DEBUG"
+	ca_ttl = "1h"
+	default_svid_ttl = "1h"
+}
+
+plugins {
+	DataStore "sql" {
+		plugin_data {
+			database_type = "sqlite3"
+			connection_string = "/opt/spire/data/server/datastore.sqlite3"
+		}
+	}
+	NodeAttestor "x509pop" {
+		plugin_data {
+			ca_bundle_path = "/opt/spire/conf/server/agent-cacert.pem"
+		}
+	}
+	KeyManager "memory" {
+		plugin_data = {}
+	}
+ 	UpstreamAuthority "spire" {
+ 	    plugin_data = {
+ 	        server_address      = "root-server"
+ 	        server_port         = 8081
+ 	        workload_api_socket = "/opt/spire/sockets/workload_api.sock"
```

Comment:
can you add comments to make it easier to understand that this socket si the one we shared (agent-root socket)

Reply:
I tried to keep config files clean in favor of following the README.md with all the instructions. 
On the tutorial I've linked this files and also copied the config part and pasted on the text to make it visible. The text also includes a link to the plugin documentation.
I prefer the reader going to the README.md file to get the details.

Said that, I would like to know what @sanderson042 thinks here. He can resolve this with his vote.

Reply:
Hmm, this is a difficult decision. I'd suggest that since some of the config files (like docker-compose.yaml) have comments, it's ok to put in a few more strategic comments. By strategic I mean in selected places like this where Marcos thinks they are helpful. I'm open to my mind being changed if Andrés does not agree. :-)